docs/deployment_policy.md

```diff
@@ -34,12 +34,19 @@ Following are the fields in the JSON representation of a deployment policy:
   - `inputs`: A list of service variables to set.
     - `name`: The name of the variable. This is the same as a variable name found in `userInputs` as defined [here](./service_def.md).
     - `value`: The value to be assigned to the variable. Service variables are typed as described in `userInputs` defined [here](./service_def.md).
+- `secretBinding`: This section is used to bind secret names defined in the servcie with the secret names in the vault. The secret value will be retrived from the vault and passed to the service container at the deployment time. The secret value is used by the service container to access other applications.
+  - `serviceUrl`: The name of the service. It can be the top level services defined in the `services` attribute or their dependent services. This is the same value as found in the `url` field [here](./service_def.md).
+  - `serviceOrgid`: The organization in which the service in `serviceUrl` is defined.
+  - `serviceArch`: The hardware architecture of the service in `serviceUrl`, or `*` to indicate any compatible architecture. This is the same value as found in the `arch` field [here](./service_def.md).
+  - `serviceVersionRange`: A version range indicating the set of service versions to which this secret binding should be applied.
+  - `secrets`: A list of secret bindings. Each elelment is a map of string keyed by the name of the secret in the service. The value is the name of the secret in the vault. The valid formats for the vault secret names are: `<secretname>` for the organization level secret; `user/<username>/<secretname>` for the user level secret. The fully qualified vault secret name will be the the value defined here preceded by `openhorizon/<orgname>/` where the `<orgname>` is the organzation name of this deployment policy. However, it is invalid to specify the fully qualified name here.
```

Comment:
more vault references. 
Also, please delete the sentence that starts "The fully qualified vault secret name will be the the value defined here preceded by `openhorizon/<orgname>/` where the `<orgname>` is the organzation name of this deployment policy. However, it is invalid to specify the fully qualified name here."

Reply:
done